src/domain/entities/team.ts

```diff
@@ -5,12 +5,12 @@ export enum MemberRole {
   /**
    * Team member can read and write notes
    */
-  read = 'read',
+  read = 0,
```

Comment:
not clear what these 0s and 1s mean. Maybe use enum for that?

Reply:
i think, it is done

Reply:
sorry, my bad, everything was ok in the first place, i didn't notice it was already in enum

Reply:
is it difficult to revert the change?

Reply:
no

Reply:
done